zuul-core/src/main/java/com/netflix/zuul/netty/server/Server.java

```diff
@@ -242,31 +277,47 @@ private void initializeTransport()
             ThreadFactory workerThreadFactory = new CategorizedThreadFactory(name + "-ClientToZuulWorker");
             Executor workerExecutor = new ThreadPerTaskExecutor(workerThreadFactory);
 
-            if (USE_EPOLL.get()) {
+            Map<ChannelOption, Object> extraOptions = new HashMap<>();
+            boolean useNio = forceNio.get();
```

Comment:
Do we want the ability to force other types? i.e. the `USE_EPOLL` flag.

Reply:
The DynamicProperties lack the ability to see if they were explicitly set.  `USE_EPOLL` can't be used to see if the caller explicitly didn't want epoll, or if they just didn't care.   The forceNio var is there to re-expose the ability to fall back to a safe transport in case we need it during a production emergency.